assets/src/dashboard/app/api/test/useStoryApi.js

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import { reshapeStoryObject } from '../useStoryApi';
+
+jest.mock('../wpAdapter', () => ({
+  get: () =>
+    Promise.resolve({
+      headers: {
+        get: () => '1',
+      },
+      json: () =>
+        Promise.resolve([
+          {
+            id: 123,
+            status: 'published',
+            title: { rendered: 'Carlos', raw: 'Carlos' },
+            story_data: { pages: [{ id: 1, elements: [] }] },
+            modified: '1970-01-01T00:00:00.000Z',
+          },
+        ]),
+    }),
+  post: (path, { data }) =>
+    Promise.resolve({
+      id: data.id,
+      status: 'published',
+      title: { rendered: data.title, raw: data.title },
+      story_data: { pages: [{ id: 1, elements: [] }] },
+      modified: '1970-01-01T00:00:00.000Z',
+    }),
+}));
```

Comment:
While it doesn’t hurt I do not think we need the jest mock in this file.

Reply:
thanks, missed this when i split up the apiProvider tests!